### PR TITLE
Merge ChanJianHao-UG-Workspace

### PR DESCRIPTION
**Pull request by ChanJianHao**
Update User Guide.

We still got to come up with a new descriptor for our application, given our target audience shift.

### CODE REVIEW

**Review by OngDeZhi — APPROVED** [review]
LGTM :)

File: docs/UserGuide.md [hunk 1]
```diff
@@ -8,34 +8,33 @@
...
-<br/>&nbsp;3.8 [View all anime in watchlist: `view`](#38-view-all-anime-in-watchlist-view)
-<br/>&nbsp;3.9 [Bookmark an Anime: `bookmark`](#39-bookmark-an-anime-bookmark)
-<br/>&nbsp;3.10 [Search: `search`](#310-search-search)
-<br/>&nbsp;3.11 [View the information of an Anime: `info`](#311-view-the-information-of-an-anime-info)
-<br/>&nbsp;3.12 [Estimate time need to translate script: `estimate`](#312-estimate-time-needed-to-translate-script-estimate)
-<br/>&nbsp;3.13 [Exit AniChan: `exit`](#313-exit-anichan-exit)
-<br/>&nbsp;3.14 [Saving and loading data](#314-saving-and-loading-data)
+<br/>&nbsp;3.2 [Workspace management: `workspace`](#32-workspace-management-workspace)
+<br/>&nbsp;3.4 [Browse through all Anime: `browse`](#33-browse-through-all-anime-browse)
+<br/>&nbsp;3.5 [Watchlist management: `watchlist`](#34-watchlist-management-watchlist)
```
Comment (EyoWeiChin) [inline docs/UserGuide.md:13]: It seems that there is a mistake in the numbering here

**Review by EyoWeiChin — COMMENTED** [review]
I apologise for the delay in the review. Thanks for working on the UG!

File: docs/UserGuide.md (hunk 1 above)
Comment (ChanJianHao) [reply docs/UserGuide.md:13]: You're right, shall leave the fix for the next update!

**Comment by ChanJianHao** [comment]
> I apologise for the delay in the review. Thanks for working on the UG!

No worries, have a good weekend!

**Pull request merged**